src/index.css

```diff
@@ -56,6 +59,10 @@ body {
     )
     0 0/3rem 3rem;
 }
+path {
```

Comment:
I've noticed, that we use those properties for almost every svg file. Here is the comparison, nothing changed, so my theory is true :)
![image](https://user-images.githubusercontent.com/49658988/187162790-9f7d26b3-c345-48a1-bc99-6cd7734a256c.png)

https://user-images.githubusercontent.com/49658988/187162832-9f8bd4f3-2232-4f57-9f6e-4f79fb314bb3.mp4